packages/ui/src/server/controllers/login.ts

```diff
@@ -41,7 +40,22 @@ export async function handleLogin(req: Request, res: Response) {
                     res,
                     response,
                     undefined,
-                    (headers) => removeSecureFlagIfOriginInsecure(req, headers),
+                    (headers: Record<string, string[]>) => {
+                        if (headers['set-cookie']) {
+                            headers['set-cookie'] = headers['set-cookie'].map((item) => {
+                                if (item.startsWith(YT_CYPRESS_COOKIE_NAME)) {
+                                    return item.replace(
+                                        YT_CYPRESS_COOKIE_NAME,
+                                        `${ytAuthCluster}:${YT_CYPRESS_COOKIE_NAME}`,
+                                    );
```

Comment:
It looks like YT_CYPRESS_COOKIE_NAME will never passed to browser, but it required for some requests from browsers depending on UI configuration.